Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -92,7 +100,7 @@ public function normalize($object, $format = null, array $context = array())
                 '@context' => $this->contextBuilder->getResourceContext($resource, $context),
                 '@id' => $context['request_uri'],
             ];
-            list($parts, $parameters) = $this->parseRequestUri($context['request_uri']);
+            $uri = $this->parseRequestUri($context['request_uri']);
```

Comment:
If prefer avoiding having hard dependency such this one (we are in the process of decreasing the coupling to Symfony too).